autoload/ctrlsf.vim

```diff
@@ -60,16 +68,16 @@ func! ctrlsf#Search(args) abort
 
     let s:current_query = args
 
-    call s:ExecSearch(s:current_query)
+    call s:ExecSearch(s:current_query, a:only_location)
 endf
 
 " Update()
 "
-func! ctrlsf#Update() abort
+func! ctrlsf#Update(only_location) abort
```

Comment:
Let `Update` remember `only_location` like very latest command stored in `s:current_query`, I think we can remove `only_location` from argument list.